src/templates/meeting-room-schedule/meeting-room-schedule.js

```diff
@@ -0,0 +1,134 @@
+import { React, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+import localeDa from 'dayjs/locale/da';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import './meeting-room-schedule.scss';
+
+/**
+ * Meeting room schedule component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The meeting room schedule content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function MeetingRoomSchedule({ content }) {
+  dayjs.extend(localizedFormat);
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const { backgroundColor, events, title, metaData, textAlign, backgroundImage } = content;
+  const [translations, setTranslations] = useState();
+  const occupiedText = content.occupiedText ? (
+    content.occupiedText
+  ) : (
+    <FormattedMessage id="occupiedText" defaultMessage="occupiedText" />
+  );
+  const rootClasses = `template-meeting-room-schedule ${textAlign}`;
```

Comment:
I think thise section is a bit messy. I would prefer them to be split into sections to increase readability: props, states, others. For example:
```
const { backgroundColor, events, title, metaData, textAlign, backgroundImage } = content;

const [currentDate, setCurrentDate] = useState(new Date());
const [translations, setTranslations] = useState(); 

const occupiedText = content.occupiedText ? (
  content.occupiedText
) : (
  <FormattedMessage id="occupiedText" defaultMessage="occupiedText">
);  

const rootClasses = `template-meeting-room-schedule ${textAlign}`;
```